### DISCUSSION TITLE
service-worker quasar

### DISCUSSION THREAD

**Discussion by RihardXXX** [quasarframework/quasar#10365] [CLI - SSR mode]
Good day!!!
Please tell me, my application is written in quasar.
I cannot set up to work with the service worker. Very little is written in the documentation.
I need to cache the stub page on the server in ssr mode (npm run ssr). So that when a person visits the site, the browser from the cache takes a stub and shows it.
My settings

**Comment by ontwikkelfabriek** [comment]
@RihardXXX 


That's a lot of caches. My settings... ? Care to elaborate a bit more about what you want?
